### DISCUSSION TITLE
More Width / height options (vh-50, w-5, w-10...)

### DISCUSSION THREAD

**Discussion by BenoitAdam** [twbs/bootstrap#35764] [Ideas] [answered]
Hey,

Using bootstrap since 2 years now,

I would really like to have more height/width options

Currently Width and height have only these :
w-25 w-50 w-75

I wish I could have w-5,w-10, w-15, etc...

Same for vh that only have 100

I wish I could have vh-25, vh-50 and vh-75

I think those would really help.

**Comment by mdo** [comment, marked as answer]
Column classes are meant to help here as there are 12 options and they're responsive. This was an intentional decision in v5.

**Reply by BenoitAdam** [reply]
That's not what I'm talking about
